noir-projects/noir-contracts/contracts/token_contract/src/test/transfer.nr

```diff
@@ -0,0 +1,64 @@
+use crate::test::utils;
```

Comment:
I just copied these test cases from `transfer_private` so does not need to be reviewed.